qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -261,6 +263,17 @@ def z(self):
     def z(self, val):
         self._z[:] = val
 
+    @property
+    def array(self):
```

Comment:
Ah I see you added this method directly to PauliList instead of Clifford. In that case it should probably be added to `BasePauli` so it applies to regular `Pauli` as well. 

It might still be worth adding the `array` method to Clifford` as suggested, which can call this method.